src/ucp/dt/dt_struct.h

```diff
@@ -51,6 +53,7 @@ typedef struct ucp_dt_struct {
     size_t len, step_len, depth;
     size_t desc_count;
     size_t rep_count;
+    ucp_dt_reg_t contig_mem;
```

Comment:
would need to save it in the `state`, struct does not represent a single registration - the same struct can be registered for different memory addresses

Reply:
Yes, I figured that out. Haven’t yet implemented

Reply:
See 840ac54 on how I was planning to approach this